Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM        node:14-alpine
+ARG         SORA_ENDPOINT
+ENV         TZ utc
+RUN         apk add --no-cache tini git
+ENTRYPOINT  ["/sbin/tini", "--", "subql-node"]
+RUN         npm i -g @subql/node
+WORKDIR     /app
+COPY        . .
+RUN         [ -z "$SORA_ENDPOINT" ] || sed -i "s framenode-2.s2.dev $SORA_ENDPOINT " project.yaml
```

Comment:
you don't need to edit this file inside the container, we can mount it when starting the container.
```
docker run -v  file:/app/project.yaml
```